onyx.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<manifest>
+    <project name="CyanogenMod/android_device_oppo_common" path="device/oppo/common" remote="github" revision="ec34d17cd81afd459071e2f6e663b1d65338257e" />
+    <project name="CyanogenMod/android_device_qcom_common" path="device/qcom/common" remote="github" revision="d51a421eb303a7eb4917902cb133a662e71f3916" />
+    <project name="taaem/android_vendor_oneplus_onyx" path="vendor/oneplus/onyx" remote="github" revision="master" />
```

Comment:
There is no CM repo for that :/ so i will leave it that way, don't know if it is actually required for building tough.